pandas/io/tests/test_clipboard.py

```diff
@@ -115,3 +119,16 @@ def test_read_clipboard_infer_excel(self):
         exp = pd.read_clipboard()
```

Comment:
can you add tests for other issues that we mentioned (or attach the issue number to an existing tests as appropriate)